virttest/qemu_monitor.py

```diff
@@ -222,7 +222,7 @@ def __setstate__(self, state):
     def __getinitargs__(self):
         # Save some information when pickling -- will be passed to the
         # constructor upon unpickling
-        return self.vm, self.name, self.filename
+        return self.vm, self.name, self.filename, True
 
```

Comment:
You set the default of suppress_exception to False.
but the getinitargs return True.
Is it OK? I'm not sure ... Just looks odd.